pkg/model/stack.go

```diff
@@ -90,6 +90,7 @@ type Service struct {
 	Healtcheck      *HealthCheck          `yaml:"healthcheck,omitempty"`
 	User            *StackSecurityContext `yaml:"user,omitempty"`
 
+	// Fields only for okteto stacks
```

Comment:
What does this mean? 
If it means that these fields are only used for stacks, should we create 2 structs for this? one for the `Service`, and a wrapper `StackService` which has the `Service` and also this fields?

Reply:
I like the idea. It may change the way in which these fields are accessed.

Reply:
Yes, We should read each manifest (okteto stacks and compose) in a way that it doesn't collide with the other and translate them to a common struct to work afterward. I think that should be covered on  https://github.com/okteto/okteto/issues/2219